pgtrigger/core.py

```diff
@@ -800,12 +837,22 @@ def _render_value():
         '''
 
 
-def get(*uris):
+def get(*uris, database=None):
     """
-    Get triggers matching URIs or all triggers registered to models
+    Get triggers matching URIs or all triggers registered to models.
+    If a database is provided, will only enable triggers
+    registered to a particular database.
 
     A URI is in the format of "app_label.model_name:trigger_name"
     """
+    if database and uris:
+        raise ValueError('Cannot supply both trigger URIs and a database')
```

Comment:
I've usually had somewhat of a hard time finding the right built-in exceptions to raise in various circumstances - I'm fine using ValueError here - but was curious if you thought `RuntimeError` would be applicable?

Reply:
Yea I have had a difficult time too. And finding answers like these https://stackoverflow.com/questions/256222/which-exception-should-i-raise-on-bad-illegal-argument-combinations-in-python does not help add any clarity.

In that thread, there were a couple answers that suggested using a ValueError for invalid argument combinations. Others suggested TypeError and then others suggested a custom exception. I have just always used ValueError when the values in the arguments are invalid or when the combination of arguments are invalid. It's just a style thing for me.

I've always used RuntimeError as a catch all when something goes wrong in the middle of running a function

Reply:
That sounds totally fine to me! I read thru python's docs as well on exceptions, and there's no clear answer there.